modules/charon-utils/src/main/java/org/wso2/charon3/utils/usermanager/InMemoryUserManager.java

```diff
@@ -60,18 +57,16 @@ public User createUser(User user, Map<String, Boolean> map)
     }
 
     @Override
-    public User getUser(String id, Map<String, Boolean> map)
```

Comment:
Seems like you have removed UserManager Interface and added ResourceHandler interface. We are not recommending to remove the public interfaces as we need to preserve the backward compatibility as the other users may depend on those interfaces.